docs/tla/BackupCol.tex

```diff
@@ -0,0 +1,1106 @@
+\batchmode %% Suppresses most terminal output.
```

Comment:
I think having just `.tla` file is enough here to produce a PDF document

Reply:
Ok. I thought it can be useful to prepare TR later.